vcpkg/contributing/vcpkg_catalog_release_process.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Microsoft/vcpkg Catalog Release Process
+description: Internal process guide for creating a new curated catalog release of vcpkg.
+ms.date: 11/30/2022
+ROBOTS: NOINDEX
+---
+
+# Microsoft/vcpkg catalog release process
```

Comment:
I still think this content should not be visible on learn pages at all.

Reply:
Learn policy is to almost never delete content if possible. What we're doing is archiving it, which will warn users that the content is not being maintained. They would also only be able to access it with a direct URL, but otherwise won't be indexed anywhere.